pkg/odo/util/cmdutils.go

```diff
@@ -136,7 +136,7 @@ func PrintComponentInfo(client *occlient.Client, currentComponentName string, co
 
 			// Gather the output
 			for _, componentURL := range componentDesc.Spec.URL {
-				url := urls.Get(componentURL)
+				url, _ := urls.Get(componentURL)
```

Comment:
please don't ignore the error. If the function returns a error, this might cause a panic as the returned URL will be empty.

Reply:
got it , thanks